browser/brave_vpn/win/brave_vpn_wireguard_service/status_tray/status_tray_runner.cc

```diff
@@ -116,8 +116,10 @@ bool StatusTrayRunner::IsVPNConnected() const {
 void StatusTrayRunner::ConnectVPN() {
   if (IsWireguardActive()) {
     wireguard::EnableBraveVpnWireguardService(
-        "", base::BindOnce(&StatusTrayRunner::OnConnected,
-                           weak_factory_.GetWeakPtr()));
+        // TODO(bsclifton): fix me. Why was empty string working before?
```

Comment:
Need to get official binaries and test this. I can't build locally since it requires signing to run WireGuard

Reply:
OK this does not work as-is. Will need to dig in more.

Reply:
OK found why it works. If empty is passed, the last used config will be read/used.
https://github.com/brave/brave-core/blob/2e7b0028082ed35fe0f53f89ab5dd0df244935e6/browser/brave_vpn/win/brave_vpn_wireguard_service/service/wireguard_tunnel_service.cc#L146-L148